README.md

```diff
@@ -51,6 +51,18 @@ Community (college) maintained list of Algorithms and Data Structures implementa
 | [Stack](https://en.wikipedia.org/wiki/Stack_(abstract_data_type)) | | | [\[X\]](stack/Stack.java) | [\[X\]](stack/stack.py) | [\[X\]](stack/stack.go) | [\[X\]](stack/stack.js) |
 
 
+## How to run them
+| Language | Steps |
+|:--------------|:----------------|
+| C |`gcc [filename.c]`<br>`./filename.out  # unix`<br>`./filename.exe  # windows`|
+| CPP |`g++ [filename.cpp]`<br>`./filename.out # unix`<br>`filename.exe # windows`|
+| Java |`javac [filename.java]`<br>`java filename # windows # unix`|
+| Python |`# for python version < 3`<br>`python [filename.py]`<br>`# for python version > 3`<br>`python3 [filename.py]`<br>`# windows # unix`|
```

Comment:
It would be better to have multi line code boxes here (triple ticks). Single line `code boxes` look ugly in this context. 
But that might not be possible in markdown table. So use HTML tables. 
Use
```
<table>
<tr>..<td>.. 
</table>
```